### DISCUSSION TITLE
Volunteer help on git Repo

### DISCUSSION THREAD

**Discussion by abarberenaCPDS** [GerryDazoo/Slinger#270] [Ideas]
Hey @GerryDazoo 👋  , your work inspired me to volunteer helping you with your repo. 🚀

I can assist with the following github-administrative tasks: 🛠️ 

- Formatting your ReadMe file
- Tagging your releases
- Assist in creating branches
- Organizing Pull requests and request your approval to publishing and/or integration

Feel free to reach out,

Abe

**Comment by GerryDazoo** [comment]
Email me at gerry@dazoo.net. And we can see how this might work.
